src/plugins/regions.ts

```diff
@@ -57,6 +58,8 @@ export type RegionParams = {
   minLength?: number
   /** Max length when resizing (in seconds) */
   maxLength?: number
+  /** The id of the channel */
+  channelId?: number
```

Comment:
In v6, this param was called `channelIdx` which was technically more correct because it's a [channel's index](https://developer.mozilla.org/en-US/docs/Web/API/AudioBuffer/getChannelData), not an id.
Should we call it `channelIndex` maybe?

Reply:
I agree with you, for some reason I didn't realize it was an index instead of an id 🙈